src/epic.rs

```diff
@@ -55,7 +55,7 @@ pub static EPICS: &[Epic] = &[
         title: "Cycle of Kumarbi",
         wiki: Wiki::Same,
```

Comment:
This should be `Wiki::None` as Wikipedia doesn't have an article called "Cycle of Kumarbi." There's an article about the god Kumarbi (https://en.wikipedia.org/wiki/Kumarbi) but not the epic(s?) "Cycle of Kumarbi"